packages/experiments/src/components/Button/MenuButton/examples/MenuButton.Example.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { MenuButton, IMenuButtonProps } from '@uifabric/experiments';
+import { Stack, Text } from 'office-ui-fabric-react';
+
+const menuItems = [{ key: 'a', name: 'Item a' }, { key: 'b', name: 'Item b' }];
+const buttonMenu: IMenuButtonProps['menu'] = render => render((MenuType, props) => <MenuType {...props} items={menuItems} />);
+
+const sectionGap = 32;
+const headingGap = 16;
+const buttonGap = 12;
```

Comment:
Hmm, these are duplicated 3 times. Will the styling of buttons appear inconsistent if they diverge? Should they be consolidated somehow?

Reply:
The styling of the buttons wouldn't appear inconsistent, this would just affect how separated the buttons are from each other, but it won't affect the styling of the actual buttons.

We could try to consolidate them but I don't know if it's worth the effort.